tardis/plasma/electron_distribution/base.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+
+from astropy import units as u
+
+
+@dataclass
+class ElectronDistribution:
+    """Electron temperature and density distribution.
+
+    temperature : Quantity
+        Electron temperatures.
+    number_density : Quantity
```

Comment:
Being that we often just take .value of these quantities, is it worth specifying that these should probably be in cgs? Or is that taken care of elsewhere?

Reply:
You're correct, I think CGS specification is reasonable